app/code/Magento/NewRelicReporting/Plugin/StatePlugin.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
```

Comment:
if you use strict type, would be great to use return types as well

Reply:
:+1: